modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -923,6 +923,20 @@ The product's default behavior of using a cost-based IDP search algorithm when c
 For more information, see link:https://neo4j.com/docs/cypher-manual/current/query-tuning/query-options/#cypher-planner[Cypher manual -> Cypher planner].
 ====
 
+.Using the unicode \u0085 in an unescaped identifier
+====
+Query::
++
+[source,cypher]
+----
+RETURN 1 as my\u0085identifier
+----
+Description of the returned code::
+The unicode `\\u0061` is deprecated in unescaped identifiers.
```

Comment:
I somehow can't understand what is deprecated, as the Unicode in the example is \u0085; in the query, it is `my\u0085identifier`, and in the description - `\\u0061`. 🙈

Reply:
Right....copy paste mistake, it should be \u0085, with a single backslash.